bench/benny.ts

```diff
@@ -91,139 +85,9 @@ async function makeSuite(testCase: Testcase): Promise<void> {
 }
 
 async function main() {
-  await makeSuite({
-    name: "int8 array",
```

Comment:
What happened to this test case in the new version?

Reply:
Oops those got lost in the shuffle. I've added this back to `testCases.ts`.